questions.md

```diff
@@ -0,0 +1,8 @@
+#Additional Questions
+
+* Here is where I can list any additional questions that I have
+
+1. I'm wondering what problems I may run into in the classroom
+2. What will I need to train my students on
+3. Will they enjoy this
+4. How much extra work this be for me
```

Comment:
With some pre-semester planning, we'll bet your grading will be faster, and more students will pass. That was the case for one teacher at University of Ulster: https://github.blog/2019-02-12-how-github-classroom-and-travis-ci-improved-students-grades/